sys-cluster/teleport/teleport-2.0.6.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils golang-build systemd user
+
+DESCRIPTION="Modern SSH server for teams managing distributed infrastructure"
+HOMEPAGE="https://gravitational.com/teleport"
+
+EGO_PN="github.com/gravitational/${PN}/..."
+
+if [ ${PV} == "9999" ] ; then
+	inherit git-r3 golang-vcs
+	EGIT_REPO_URI="https://github.com/gravitational/${PN}.git"
+else
+	inherit golang-vcs-snapshot
+	SRC_URI="https://github.com/gravitational/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE="etcd systemd"
+
+DEPEND="
+	>=dev-lang/go-1.7
+	app-arch/zip"
+RDEPEND="
+	systemd? ( sys-apps/systemd )
```

Comment:
1. Why do you need this dependency? Does it actually use systemd in some way, or it is just because of the unit file? If the latter, remove the flag, the dependency and install it unconditionally.
2. Sort dependencies (including USE conditionals) lexically.